v2g-liberty-cards/src/ping-card.ts

```diff
@@ -1,10 +1,13 @@
-import { css, html, LitElement, nothing } from 'lit';
+import { html, LitElement, nothing } from 'lit';
 import { customElement, state } from 'lit/decorators';
 import { HomeAssistant, LovelaceCardConfig } from 'custom-card-helpers';
 
 import { callFunction } from './util/appdaemon';
 import { partial } from './util/translate';
 
+// Import the snackbar from MWC
+import '@material/mwc-snackbar';
```

Comment:
This will increase the size of the generated file.
I would expect this component to be loaded already by HomeAssistant, so if we can avoid loading it here, that would save some space in the v2g-liberty-cards.js .

Reply:
Good call! The snackbar is available in HA for certain.
Would you have a suggestion on where/how to find out how to use that one?

Reply:
I would expect that we can just use it like you do, without explicitly loading it ourselves.
Since HA already loads it, it should still work.